schemas/dc-library.xsd

```diff
@@ -391,35 +391,31 @@
     <xs:complexType>
       <xs:sequence>
         <xs:choice>
-          <xs:element ref="tocContainer" />
-          <xs:element ref="tocSection" maxOccurs="unbounded" />
+          <xs:element name="container" type="tocContainer" />
+          <xs:element name="section" type="tocSection" maxOccurs="unbounded" />
```

Comment:
I think this maxOccurs should be on the choice element, same as below, so there can be as many containers or sections as want inside a toc.